app/services/sufia/workflow/abstract_notification.rb

```diff
@@ -32,7 +32,7 @@ def message
       private
 
         def users_to_notify
-          recipients[:to] + recipients[:cc]
+          recipients['to'] + recipients['cc']
```

Comment:
@revgum To expound on that... the `cc` field is optional, and the JSON validator assumes `to` and `cc` (if present) are arrays: https://github.com/projecthydra/curation_concerns/blob/master/app/services/curation_concerns/workflow/workflow_schema.rb#L48